### DISCUSSION TITLE
配置了全局cesiumPath, 但还是去unpkg.com取Cesium.js

### DISCUSSION THREAD

**Discussion by obamin** [zouyaoji/vue-cesium#255] [Q&A] [answered]
![peizhixinxi](https://user-images.githubusercontent.com/36915753/184464771-05867e5b-d786-4b18-a13f-894b0700f32d.png)
我已经app.use(VueCesium, {
    cesiumPath: '/Cesium/Cesium.js',
  }), 但还是去unpkg.com取Cesium.js, 还有哪里需要设置的吗?

**Comment by zouyaoji** [comment, marked as answer]
vc-viewer上面你是不是还包裹一个vc-config-provider

**Reply by obamin** [reply]
是的, 现在去掉就正常了, 感谢

**Reply by starsfusu** [reply]
这个资源在哪里下载啊
